scripts/synchronize_panes.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# setting the locale, some users have issues with different locales, this forces the correct one
+export LC_ALL=en_US.UTF-8
+
+current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $current_dir/utils.sh
+
+get_synchronize_panes_status() {
+  current_synchronize_panes_status=$(get_tmux_window_option "synchronize-panes" "off")
+  echo $current_synchronize_panes_status
+}
+
+main()
+{
+  # storing the refresh rate in the variable RATE, default is 5
+  RATE=$(get_tmux_option "@dracula-refresh-rate" 5)
+  synchronize_panes_label=$(get_tmux_option "@dracula-synchronize-panes-label" "Sync")
+  synchronize_panes_status=$(get_synchronize_panes_status)
```

Comment:
Please put these in dracula.sh and pass the variables through to the script.

Reply:
Sorry, only the top one, not the bottom.

Reply:
Hi @ethancedwards8, i moved the get_tmux_option to the dracula.sh as you wished.